e2eAIOK/DeNas/train.py

```diff
@@ -63,7 +68,13 @@ def main(cfg):
         metric = bert_create_metric(cfg)
         trainer = BERTTrainer(cfg, model, train_dataloader, eval_dataloader, other_data, optimizer, criterion, scheduler, metric)
     elif cfg.domain == 'asr':
-        model = ModelBuilderASRDeNas(cfg).create_model()
```

Comment:
in the train.py, you shouldn't enable pruner per domain. Instead, should be check pruner config?? i.e., built-in model, other torch models also supported.